### PR TITLE
For --google_style, break between ( and long condition expressions

### PR DESCRIPTION
**Pull request by nreid260**
Fixes https://github.com/facebookincubator/ktfmt/issues/319

### CODE REVIEW

**Comment by cgrushko** [comment]
Importing for easier review

File: core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt [hunk 1]
```diff
@@ -698,6 +698,31 @@ class KotlinInputAstVisitor(
...
+        }
+      } else {
+        builder.block(ZERO) {
+          visit(condition)
+        }
+      }
+    }
+    builder.token(")")
+  }
+
```
Comment (strulovich) [inline core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt:725]: We organize methods by chronological order if possible, so caller before callee. Could you move these below all callers?

File: core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt [hunk 2]
```diff
@@ -698,6 +698,31 @@ class KotlinInputAstVisitor(
     return extractCallExpression(this)?.lambdaArguments?.isNotEmpty() ?: false
   }
 
+  private fun emitKeywordWithCondition(keyword: String, condition: KtExpression?) {
```
Comment (strulovich) [inline core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt:701]: Please add a KDoc. 

We usually state example of the content this parses, so something alike:

```
/** Emits a key word followed by a condition, e.g. `if (b)` or `while (c <d )` */
```

**Review by strulovich — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt (hunk 2 above)
Comment (nreid260) [reply core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt]: Done

File: core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt (hunk 1 above)
Comment (nreid260) [reply core/src/main/java/com/facebook/ktfmt/format/KotlinInputAstVisitor.kt]: Done

**Review by strulovich — APPROVED** [review]
Thanks!

**Pull request closed without merging**